### PR TITLE
[TACHYON-588] Adding a backlink to the master web server to worker's navigation bar.

### PR DESCRIPTION
**Pull request by yuzhu**
Modified master and worker jsp files to include a newly created header servlet
instead of the old header jsp. Additional information about the master is added.

https://tachyon.atlassian.net/browse/TACHYON-588

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist.

**Comment by haoyuan** [comment]
Thanks @yuzhu !

Could you please copy paste a page screenshot here?

**Comment by gpang** [comment]
Jenkins, test this please.

**Comment by yuzhu** [comment]
screenshot 2015-07-09 18 20 27

Screenshot attached, the latest patches fixes the special case where master is created using a hostname "localhost", which is not accessible from external browsers.

**Comment by apc999** [comment]
+1 LGTM

File: servers/src/main/webapp/header.jsp [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<%@ page import="java.util.*" %>
+<%@ page import="tachyon.web.*" %>
```
Comment (calvinjia) [inline servers/src/main/webapp/header.jsp]: What's used in the web package?

Comment (yuzhu) [inline servers/src/main/webapp/header.jsp]: removed..

File: servers/src/main/webapp/worker/general.jsp [hunk 2]
```diff
@@ -16,7 +16,7 @@
 <script src="../js/bootstrap.min.js"></script>
 <div class="container-fluid">
   <% request.setAttribute("useWorkerHeader", "1"); %>
-  <jsp:include page="../header.jsp" />
+  <jsp:include page="/header"/>
```
Comment (calvinjia) [inline servers/src/main/webapp/worker/general.jsp]: Missing a space after the `/`?

**Comment by calvinjia** [comment]
Minor style issue, but overall looks good to me, thanks.

File: servers/src/main/webapp/worker/general.jsp (hunk 2 above)
Comment (yuzhu) [inline servers/src/main/webapp/worker/general.jsp]: fixed

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by haoyuan** [comment]
Thanks!

**Pull request merged**